app/helpers/settings_helper.rb

```diff
@@ -105,4 +105,14 @@ def compact_account_link_to(account)
       safe_join([image_tag(account.avatar.url, width: 15, height: 15, alt: display_name(account), class: 'avatar'), content_tag(:span, account.acct, class: 'username')], ' ')
     end
   end
+
+  def picture_hint(hint, picture)
+    link = link_to t('generic.delete'), settings_profile_picture_path(picture.name.to_s), data: { method: :delete }
+
+    if picture.original_filename.nil?
+      hint
+    else
+      safe_join([hint, link], '<br/>'.html_safe)
+    end
```

Comment:
I'd move the `link` thing in the `else` statement.

Reply:
Good point, fixed.